avocado/utils/process.py

```diff
@@ -492,9 +507,13 @@ def _init_subprocess(self):
             else:
                 cmd = self.cmd
             try:
+                if self.allow_output_check == 'combined':
+                    stderr = subprocess.STDOUT
+                else:
+                    stderr = subprocess.PIPE
```

Comment:
ok, now I see why it's not possible to have both and stderr+stdout at the same time. Fine.